cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/TopologyExport.java

```diff
@@ -213,6 +213,24 @@ private static void writeDcTerminals(Network network, String cimNamespace, XMLSt
             writeDcTerminals(line, line.getConverterStation1(), 1, cimNamespace, writer, context);
             writeDcTerminals(line, line.getConverterStation2(), 2, cimNamespace, writer, context);
         }
+        for (DcConnectable<?> dcConnectable : network.getDcConnectables()) {
+            for (DcTerminal dcTerminal : dcConnectable.getDcTerminals()) {
+                String className = dcConnectable instanceof AcDcConverter<?> ? CgmesNames.ACDC_CONVERTER_DC_TERMINAL : CgmesNames.DC_TERMINAL;
+                TerminalNumber terminalNumber = dcTerminal.getTerminalNumber();
+                int sequenceNumber = terminalNumber != null ? terminalNumber.getNum() : 1;
```

Comment:
`TerminalNumber` is used for `AcDcConverter`s only. For the other `DcConnectable`s, the side is used.
In this state, 2 `cim:DCTerminal` tags are generated for `#_T_DCL_1_1` (DcLine terminal 1) in `dcNetworkTpExportTest()` and none for `#_T_DCL_1_2` (DcLine terminal 2).


```suggestion
                int sequenceNumber = terminalNumber != null ? terminalNumber.getNum() : dcTerminal.getSide() != null ? dcTerminal.getSide().getNum() : 1;
```